sway/tree/container.c

```diff
@@ -1226,10 +1228,13 @@ void container_add_gaps(struct sway_container *c) {
 		}
 	}
 
-	c->current_gaps.top = c->y == ws->y ? ws->gaps_inner : 0;
-	c->current_gaps.right = ws->gaps_inner;
-	c->current_gaps.bottom = ws->gaps_inner;
-	c->current_gaps.left = c->x == ws->x ? ws->gaps_inner : 0;
+	int gaps_horizontal = MAX(0, ws->gaps_inner + MIN(0, c->width - ws->gaps_inner - MIN_SANE_W));
```

Comment:
style: 80 columns

However, I think this would be more readable as:
```c
int gaps_horizontal = MAX(0, MIN(ws->gaps_inner, c->width - MIN_SANE_W));
```

(And similar for the others)

Reply:
Yep, that does look better, thanks.